dart/lib/src/utils/sample_rate_format.dart

```diff
@@ -0,0 +1,292 @@
+import 'dart:math';
+
+import 'package:meta/meta.dart';
+
+/// Implementation details from `intl` package
+/// https://pub.dev/packages/intl
```

Comment:
@chadwhitacre I'd like to double check if this is fine, License https://github.com/dart-lang/intl/blob/master/LICENSE `BSD 3`
Should we copy the license text on top of this file?

Reply:
Yes, we actually just clarified [the policy around this🔒](https://www.notion.so/sentry/Open-Source-Legal-Policy-ac4885d265cb4d7898a01c060b061e42?pvs=4#e1e3f2040b1b4f69bc888d1e3d6b5d8c) (in https://github.com/getsentry/team-ospo/issues/120):

1. Separate file
2. Link to where we got it (attribution)
3. Include license text in file header

Reply:
@denrase lets do this then, make CI happy and merge it.